src/app/shared/dockstore.service.ts

```diff
@@ -1,13 +1,19 @@
-import { Injectable } from '@angular/core';
-import { Http } from '@angular/http';
+import {Inject, Injectable} from '@angular/core';
+import { Http, RequestMethod, URLSearchParams} from '@angular/http';
+import { AuthService } from 'ng2-ui-auth';
+import { Dockstore } from '../shared/dockstore.model';
+import { HttpService } from './http.service';
+
 
 @Injectable()
 export class DockstoreService {
   private static readonly months = [ 'Jan.', 'Feb.', 'Mar.', 'Apr.',
```

Comment:
You are right, it is unnecessary because the month array is also defined in the date service. Only the date service is using it. Thus, I think we can remove it from the dockstore service (unless we need to use it in the future). 
What do you think?